webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/TokenLifeTime.js

```diff
@@ -14,7 +14,7 @@ import LabelIcon from '../../../../../components/common/LabelIcon';
 import { sprintf, translate as __ } from '../../../../../common/I18n';
 
 const TokenLifeTime = ({ value, onChange, handleInvalidField, isLoading }) => {
-  const minValue = 1;
```

Comment:
What's the use-case for it? Why would the user want to create a registration command with an expired token?

Reply:
I think so too, I spoke with QAs, they do not use it for testing either. I think it would be safe to use '0' to represent 'unlimited' 
@pablomh what do you think?

Reply:
I agree with @stejskalleos and I think that we should avoid accepting an "expired token option" and settle on:

- x < 0 raises an error, invalid value
- x = 0 Unlimited token
- x > 0 expiration in hours

@sayan3296, opinions?

Reply:
I apologize for the late reply and that sounds fine to me as long as everyone else agrees.